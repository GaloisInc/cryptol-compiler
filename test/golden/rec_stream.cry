main = take`{10} (f (5:[64]) fibs)

fibs = xs
  where
  xs = [0,1] # [ x + y | x <- xs | y <- drop`{1} xs ]


f a ys = xs
  where
  xs = [0] # [ a + x + 1 + y | x <- xs | y <- ys ]
