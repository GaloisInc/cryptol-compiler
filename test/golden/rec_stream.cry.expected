[0x0000000000000000, 0x0000000000000006, 0x000000000000000c, 0x0000000000000012, 0x0000000000000018, 0x000000000000001e, 0x0000000000000024, 0x000000000000002a, 0x0000000000000030, 0x0000000000000036]

